src/vs/workbench/browser/parts/statusbar/media/statusbarpart.css

```diff
@@ -224,3 +224,8 @@
 	color: var(--vscode-statusBarItem-offlineHoverForeground);
 	background-color: var(--vscode-statusBarItem-offlineHoverBackground) !important;
 }
+
+.monaco-workbench .part.statusbar > .items-container > .statusbar-item[id="status.problems.off"] {
```

Comment:
For what is this?

Reply:
![image](https://github.com/microsoft/vscode/assets/54879025/87df3bcd-edea-41bf-bb2e-7d4081ef2a40)

We decided to add a little indicator in the status bar right next to the status bar problems to indicate the setting is turned off

Reply:
But can this be done from the component that owns problems?